### DISCUSSION TITLE
Allow other language as ComponentName

### DISCUSSION THREAD

**Discussion by sooohka** [vitejs/vite-plugin-react-swc#235] [Ideas]
I personally use Korean as Component name but isLikelyComponentType only allow English Capital as a component name,
due to this i can not get react-refresh to work properly.

If there's no reason, I think it's okay to test only lowercase English letters

thanks

**Comment by ArnaudBarre** [comment]
This is something that is also builtin on the react side and they made it stricter in the latest eslint plugin release: https://github.com/facebook/react/blob/bf7e210cb5672685bfe992a3b253880f5a3d47f5/packages/eslint-plugin-react-hooks/src/RulesOfHooks.js#L48-L50

I don't think it will change. You can always prefix your component with the same capital letter and then write the name in your language (like some people always put a `T` at the start of generic names)
